docs/_sass/sidebar.scss

```diff
@@ -147,6 +150,12 @@
           position: relative;
           width: fit-content;
 
+		  &.active,
+		  &.active a{
+			scroll-margin-bottom: 8rem;
```

Comment:
where you  are giving this margin-bottom

Reply:
Yes it's because the `scrollElem.scrollIntoView({ behavior: 'smooth', block: 'nearest', inline: 'start'});` property that I am using is causing the scroll to focus on the selected nav item. But the focus item is itself moving to the bottom. Therefore using `scroll-margin-bottom` for the active element will give the element itself some space from the bottom after the scroll focus. Now you might think why `8rem` its because If I give it more than that then it will mess with the scrolling of the body.